news/2025/2025-04-15-the-followpoint-azer-the-tournament-titan.md

```diff
@@ -0,0 +1,305 @@
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/2155578">Azer</a></p>
+
+When I started casting, commentary was a lot less polished, so it was easy as a knowledgeable player to step in and talk about stuff, but over time certain casters came along and really made everyone else step up their game. I don't particularly consider myself a great caster but my depth of experience still lets me bring interesting topics, and that type of casting is kinda hard to come by (most casters had great voices/pacing but not many really had high level player experience, so I slotted in the ecosystem easily)
+
+Most of my motivation to improve came from a few casters who were really good at areas I was lacking, mostly being coherent and being able to direct conversation, some names like [emanfman](https://osu.ppy.sh/users/4136150), [Dohland](https://osu.ppy.sh/users/5220511), [Bae-](https://osu.ppy.sh/users/6576972), and eventually [D I O](https://osu.ppy.sh/users/3958619) and [ChillierPear](https://osu.ppy.sh/users/9501251) would come up over time as voices that really impressed me and always felt comfortable to commentate with. I really think that within the osu! standard tournament scene, the level of casting we have is so incredibly elevated by great talent like those mentioned, and when comparing to any events recorded prior to those people improving the scene, it really shines a big spotlight at how proud we can be at the sheer amount of top tier casting talent we have in our community!
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+What does the osu! World Cup sound team do?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/2155578" style="background-image: url('/wiki/shared/avatars/Azer.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/2155578">Azer</a></p>
+
+It's not really a real team to be fair, I just like the name but it represents the small team we have in place to create osu! originals and help them flourish. For the most part it's just [mangomizer](https://osu.ppy.sh/users/1893718) and myself, but lately we have also benefited from the help of [Kry.exe](https://osu.ppy.sh/beatmaps/artists/448), who is one of very few people who hold all the criteria required to contribute to what we do. We essentially provide production feedback to artists we work with in order to create memorable experiences for world cup events. It can be music related, or suitability to gamemode or intent, it can be fixes intended for gameplay etc.
```

Comment:
```suggestion
It's not really an official team to be fair, I just like the name because it represents the small team we have in place to create osu! originals and help them flourish. For the most part it's just [mangomizer](https://osu.ppy.sh/users/1893718) and myself, but lately we have also benefited from the help of [Kry.exe](https://osu.ppy.sh/beatmaps/artists/448), who is one of very few people who hold all the criteria required to contribute to what we do. We essentially provide production feedback to artists we work with in order to create memorable experiences for world cup events. It can be music related, or suitability to gamemode or intent, it can be fixes intended for gameplay etc.
```